django/middleware/http.py

```diff
@@ -11,6 +11,7 @@ class ConditionalGetMiddleware(MiddlewareMixin):
 
     Also sets the Date and Content-Length response-headers.
     """
+
```

Comment:
The guideline is "don't include unrelated cosmetic changes in bug/feature patch."